screens/auth/Form.css

```diff
@@ -0,0 +1,58 @@
+.page-wrap {
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+}
+
+.login-form-container {
+  flex-grow: 1;
+  margin-top: 1.5rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.login-form {
+  box-shadow: var(--card-box-shadow);
+  height: max-content;
+  padding: 1.6rem;
+  width: 375px;
```

Comment:
using rem consistently is a good practice & must be followed. You may do these changes later though.

Reply:
Thanks! Surely I'll follow that.